packages/lesmis/src/components/Tabs.js

```diff
@@ -3,25 +3,46 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  *
  */
-import React from 'react';
+import React, { forwardRef } from 'react';
 import PropTypes from 'prop-types';
 import MuiContainer from '@material-ui/core/Container';
 import styled from 'styled-components';
 import MuiTabs from '@material-ui/core/Tabs';
 import Skeleton from '@material-ui/lab/Skeleton';
 import MuiTab from '@material-ui/core/Tab';
+import { NAVBAR_HEIGHT } from '../constants';
 import { FlexStart } from './Layout';
 
 const LoadingTab = () => <Skeleton width={100} height={20} style={{ marginLeft: 30 }} />;
 
-export const TabBar = styled(MuiContainer)`
+const TabBarOuterContainer = styled.div`
+  border-top: 1px solid ${props => props.theme.palette.grey['400']};
+  border-bottom: 1px solid ${props => props.theme.palette.grey['400']};
+  background: white;
+  z-index: 9999;
+`;
+
+const StickyTabBarOuterContainer = styled(TabBarOuterContainer)`
+  position: sticky;
+  top: ${NAVBAR_HEIGHT};
+`;
+
+const TabBarInnerContainer = styled(MuiContainer)`
   display: flex;
   align-items: center;
   justify-content: flex-start;
-  border-top: 1px solid ${props => props.theme.palette.grey['400']};
-  border-bottom: 1px solid ${props => props.theme.palette.grey['400']};
 `;
 
+// sticks on scroll, sitting below the nav bar
+export const StickyTabBar = forwardRef(({ children }, ref) => (
```

Comment:
I would prefer to get rid of this component and move the 2 containers inline in `DashboardReportTabView`. It makes sense if we had an encapsulated sticky tab bar component but at the moment most of the code for it is in the tab view any way. This keeps the tabs components free of the sticky logic, removes the forwardRef which could be confusing and removes the need to look between files for parts of the behaviour.

Reply:
This was a great idea - it's heaps nicer now :-)